azure-pipelines.yml

```diff
@@ -1,35 +1,44 @@
 trigger:
+- dev-it
 - master
 - dev-remote-ci
 
 jobs:
-
 - job: 'Ubuntu_16_04'
-  pool:
-    vmImage: 'Ubuntu 16.04'
-  strategy:
-    matrix:
-      Python36:
-        PYTHON_VERSION: '3.6'
+  pool: 'NNI CI GPU'
 
   steps:
-  - script: python3 -m pip install --upgrade pip setuptools
+  - script: python3 -m pip install --upgrade pip setuptools --user
     displayName: 'Install python tools'
   - script: |
       source install.sh
     displayName: 'Install nni toolkit via source code'
+  - script: |
+      python3 -m pip install sklearn --user
+      python3 -m pip install torchvision --user
+      python3 -m pip install keras --user
+      python3 -m pip install tensorflow==1.9.0 --user
```

Comment:
Suggest to install tensorflow-gpu 1.10, to be consistent with NNI official Docker image.

Reply:
Thanks, updated.